docs/resources/artifactory_virtual_conan_repository.md

```diff
@@ -0,0 +1,42 @@
+# Artifactory Virtual Conan Repository Resource
+
+Provides an Artifactory virtual repository resource, but with specific conan features. This should be preferred over the original
+one-size-fits-all `artifactory_virtual_repository`. 
+
+## Example Usage
+
+```hcl
+resource "artifactory_virtual_conan_repository" "foo-conan" {
```

Comment:
You see, I only include the things that are unique to conan - everything here seems to be just like the base repo